optaplanner-core/src/test/java/org/optaplanner/core/impl/testdata/phase/event/TestdataStepValueListener.java

```diff
@@ -0,0 +1,37 @@
+package org.optaplanner.core.impl.testdata.phase.event;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import org.optaplanner.core.impl.phase.event.PhaseLifecycleListenerAdapter;
+import org.optaplanner.core.impl.phase.scope.AbstractStepScope;
+import org.optaplanner.core.impl.solver.scope.DefaultSolverScope;
+import org.optaplanner.core.impl.testdata.domain.comparable.TestdataComparableEntity;
+import org.optaplanner.core.impl.testdata.domain.comparable.TestdataComparableSolution;
+
+public class TestdataStepValueListener extends PhaseLifecycleListenerAdapter<TestdataComparableSolution> {
+
+    private final List<String> dataConfigurations = new ArrayList<>();
```

Comment:
What's a dataConfiguration? This names doesn't seem to fit the payload.

Reply:
Changed it to workingSolutions. Does it fit the payload properly?